### ISSUE TITLE
Kundera v2.7, external configuration

### ISSUE THREAD

**Issue by bsideup** [Impetus/kundera#402]
Hi!

In current version AbstractPropertyReader reads "kundera.client.property" file as Resource ( https://github.com/impetus-opensource/Kundera/blob/4f988169b0467303726183597d48d6f82b793670/kundera-core/src/main/java/com/impetus/kundera/configure/AbstractPropertyReader.java#L90 ) :

``` java
/**
     * If property file is xml.
     * 
     * @param propertyFileName
     * @param puMetadata
     * @return
     */
    private ClientProperties onParseXML(String propertyFileName, PersistenceUnitMetadata puMetadata)
    {
        InputStream inStream = puMetadata.getClassLoader().getResourceAsStream(propertyFileName);
        if (inStream == null)
        {
            propertyFileName = KunderaCoreUtils.resolvePath(propertyFileName);
            try
            {
                inStream = new FileInputStream(new File(propertyFileName));
            }
            catch (FileNotFoundException e)
            {
                log.warn("File {} not found, Caused by ", propertyFileName);
                return null;
            }
        }
        else
        {
            xStream = getXStreamObject();
            Object o = xStream.fromXML(inStream);
            return (ClientProperties) o;
        }
        return null;
    }
```

Is it possible to read it from filesystem instead?

For example, we've different environments, such as dev, staging, prod, and we want to have database config file such as **/etc/ourApp/mongo.xml**, and it's will different on each env type, because we use different databases. 

It's very useful feature in big projects and it would be very helpful. 

Thanks!

**Comment by mevivs** [comment]
This has been fixed in 2.7.1 to read from environment variables.

https://github.com/impetus-opensource/Kundera/issues/391

-Vivek

**Comment by bsideup** [comment]
OK! Thanks!
